synapse/storage/databases/main/roommember.py

```diff
@@ -670,6 +670,28 @@ async def get_users_who_share_room_with_user(
 
         return user_who_share_room
 
+    @cached(
+        max_entries=500000,
+        cache_context=True,
+        iterable=True,
+        prune_unread_entries=False,
+    )
+    async def get_mutual_rooms_between_users(
```

Comment:
Docstring please.

Reply:
This function already has a docstring, did you mean to say it has to be more descriptive?